src/BackendEvent.ts

```diff
@@ -24,11 +24,6 @@ export interface BackendEvent {
      * The type of action generating this event
      */
     type: BackendEventType;
-    /**
-     * The identifier for the run this message is associated with
-     * This allows discarding outdated events that were delayed
-     */
-    runId: number;
```

Comment:
Comsync only handles this for its own internal callbacks, you may still want this for your own events.

Reply:
The runId was only used to filter out delayed events. The interrupts should smoothen this process so that I no longer need this filtering step.